test/SignalR.Orleans.Tests/OrleansHubLifetimeManagerTests.cs

```diff
@@ -176,24 +203,17 @@ public async Task InvokeConnectionAsync_WritesToConnections_Output()
         public async Task InvokeConnectionAsync_OnNonExistentConnection_DoesNotThrow()
         {
             var invalidConnection = "NotARealConnectionId";
-            var grain = this._fixture.ClientProvider.GetClient().GetClientGrain("MyHub", invalidConnection);
-            await grain.OnConnect(Guid.NewGuid(), "MyHub", invalidConnection);
-            var manager = new OrleansHubLifetimeManager<MyHub>(new LoggerFactory().CreateLogger<OrleansHubLifetimeManager<MyHub>>(), this._fixture.ClientProvider);
+            var grain = _fixture.ClientProvider.GetClient().GetClientGrain("MyHub", invalidConnection);
+            await grain.OnConnect(Guid.NewGuid());
+            var manager = new OrleansHubLifetimeManager<MyHub>(new LoggerFactory().CreateLogger<OrleansHubLifetimeManager<MyHub>>(), _fixture.ClientProvider);
             await manager.SendConnectionAsync(invalidConnection, "Hello", new object[] { "World" });
         }
 
-        [Fact]

```

Comment:
this test was removed as its a bit against the fix i did for group (see `InvokeGroupAsync_WhenOneDisconnected_ShouldDeliverOthers`)